src/styles/search-bar.css

```diff
@@ -0,0 +1,48 @@
+.search-bar {
+  width: 100%;
+  height: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  color: #3d4e61;
+  font-weight: 300;
+  font-family: inherit;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  outline: none;
+}
+
+.search-bar:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search-bar_big {
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.search-bar_small {
+  padding-left: 33px;
+  font-size: 14px;
+  line-height: 19px;
+}
+
+.search-bar::placeholder {
+  color: inherit;
+  font-weight: 300;
+}
+
+.search-bar_big::placeholder {
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.search-bar_small::placeholder {
+  font-size: 14px;
+  line-height: 22px;
```

Comment:
The line-height for .search-bar_small::placeholder is different from the font-size which is unusual for a placeholder. It's typically kept the same or very close to the font-size for consistency.